dashboard/app/models/pd/enrollment.rb

```diff
@@ -97,6 +101,14 @@ def in_section?
     workshop.section.students.exists?(user.id)
   end
 
+  def exit_survey_url
```

Comment:
As I said above, it's not directly related. Initially it was because I was thinking about redirecting back to the survey in case it wasn't filled out, but I decided against it. I still think it's an improvement, but I'm happy to move it into its own PR if you prefer.